instant-xml-macros/src/lib.rs

```diff
@@ -2,10 +2,13 @@ extern crate proc_macro;
 
 use proc_macro::TokenStream;
 use quote::quote;
+use std::collections::HashMap;
 use syn::{parse_macro_input, DeriveInput, Lit, Meta, NestedMeta};
 
-fn retrieve_default_namespace(input: &DeriveInput) -> Option<String> {
-    for attr in &input.attrs {
+fn retrieve_namespace_list(
+    attributes: &Vec<syn::Attribute>,
+) -> Option<syn::punctuated::Punctuated<NestedMeta, syn::token::Comma>> {
```

Comment:
Maybe this could return an `impl Iterator<Item = NestedMeta>`?

Reply:
Not relevant anymore after refactoring.
I might be missing something but it seems to be hard to do as `Nested Meta` is a local variable, as this:
[            let nested = match attr.parse_meta() {](https://github.com/InstantDomain/instant-xml/blob/1b5a5a3d6a77568768f29bc64ffc472db7714515/instant-xml-macros/src/lib.rs#L25)
is returning the `Meta` enum.